glean_parser/templates/kotlin.jinja2

```diff
@@ -10,6 +10,7 @@ Jinja2 template is not. Please file bugs! #}
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 {% macro obj_declaration(obj, suffix='', access='', lazy=False) %}
+@get:JvmName("{{ obj.name|camelize }}{{ suffix }}")
```

Comment:
While this makes sense theoretically, any chance you could provide a sample output for this? It would be great if you could also create a work in progress PR with test coverage that depends on this, to try it locally.

Reply:
Would it be enough for me to add the example here as a comment? If so, here you go:

```kotlin
// -*- mode: kotlin -*-

/*
 * AUTOGENERATED BY glean_parser.  DO NOT EDIT.
 */

/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

/* ktlint-disable no-blank-line-before-rbrace */
@file:Suppress("PackageNaming", "MaxLineLength")
package GleanMetrics

import mozilla.components.service.glean.private.HistogramType // ktlint-disable import-ordering no-unused-imports
import mozilla.components.service.glean.private.Lifetime // ktlint-disable import-ordering no-unused-imports
import mozilla.components.service.glean.private.MemoryUnit // ktlint-disable import-ordering no-unused-imports
import mozilla.components.service.glean.private.NoExtraKeys // ktlint-disable import-ordering no-unused-imports
import mozilla.components.service.glean.private.TimeUnit // ktlint-disable import-ordering no-unused-imports
import mozilla.components.service.glean.private.PingType // ktlint-disable import-ordering

internal object Pings {
    /**
     * This is a custom ping
     */
    @get:JvmName("customPing")
val customPing: PingType<NoExtraKeys> =
        PingType<NoExtraKeys>(
            includeClientId = false,
            sendIfEmpty = false,
            name = "custom-ping",
            reasonCodes = listOf()
        )

    /**
     * Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
     * incididunt ut labore et dolore magna aliqua.
     * Ut enim ad minim veniam, quis
     * nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
     * Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
     * fugiat nulla pariatur.
     */
    @get:JvmName("reallyCustomPing")
val reallyCustomPing: PingType<NoExtraKeys> =
        PingType<NoExtraKeys>(
            includeClientId = true,
            sendIfEmpty = false,
            name = "really-custom-ping",
            reasonCodes = listOf()
        )

    /**
     * This is another custom ping
     */
    @get:JvmName("customPingMightBeEmpty")
val customPingMightBeEmpty: PingType<NoExtraKeys> =
        PingType<NoExtraKeys>(
            includeClientId = true,
            sendIfEmpty = true,
            name = "custom-ping-might-be-empty",
            reasonCodes = listOf()
        )

}
```
This is the outpot for https://github.com/mozilla/glean_parser/blob/master/tests/data/pings.yaml